test.sh.in

```diff
@@ -346,6 +346,23 @@ fetchOVALog(){
     popd
 }
 
+###########################################
+#
+#  Ensure logs are dumpped even build timeout aborted
+#
+#############################################
+general_cleanup(){
+
+    set +e
+    cleanupVMs
+    generateSysLog
+    generateMongoLog
+    vnc_record_stop
+
```

Comment:
The method ```cleanupVMs``` will destroy vagrant box.
The methods ```generateSysLog``` , ```generateMongoLog``` and ```vnc_record_stop``` all need to login the vagrant.
So, please run ```cleanupVMs``` at the last step.